tensorflow/stream_executor/cuda/cudart_stub.cc

```diff
@@ -131,6 +131,13 @@ extern __host__ __device__ unsigned CUDARTAPI __cudaPushCallConfiguration(
   return func_ptr(gridDim, blockDim, sharedMem, stream);
 }
 
+extern char CUDARTAPI __cudaInitModule(void **fatCubinHandle) {
+  using FuncPtr = cudaError_t(CUDARTAPI *)(void **fatCubinHandle);
```

Comment:
return type should be char

Reply:
done. too eager with copy-paste

Reply:
I thought that might have been the case :). LG!